apps/examples/app/examples/stack/demo.tsx

```diff
@@ -0,0 +1,16 @@
+import { Stack, Box } from "@postenbring/hedwig-react";
+
+function Example() {
+  return (
+    <Stack gap="12-16" direction={{ xsmall: "column", small: "row" }}>
+      <Box>1</Box>
+      <Box>2</Box>
+      <Box>3</Box>
+    </Stack>
```

Comment:
💭 
In most cases it's better to use HStack or VStack instead of passing `direction`. The exception is when you need the direction to be responsive.